distribution/docs/src/main/resources/content/_quickstart/quickstart-configuring.adoc

```diff
@@ -14,7 +14,9 @@ Set the configurations needed to run ${branding}.
 . Follow the installer prompts for a standard installation.
 .. Click start to begin the setup process.
 .. Configure <<_guest_interceptor,guest claims attributes>> or use defaults.
-... *All users will be automatically granted these authorizations*
+... See <<Configuring Guest Access>> for more information about the Guest user.
+... *All users will be automatically granted these permissions.*
+... *Any data ingested will need to have these attributes to be accessible by Guest users.*
```

Comment:
✏️ This isn't *literally* correct when expansion and match-one/match-all come into play. It's more the other way around. The metacard must be marked in such a way that the guest user's claims permit guest access.

Reply:
Yes. Would read more like `Any data ingested that has more restrictive markings than these guest claims, will not be visible to Guest users`.
❗️ Before this bullet point, we should also say that the Guest user will not be able to ingest such data either.

Reply:
added.